### DISCUSSION TITLE
Meal plan by category and tag

### DISCUSSION THREAD

**Discussion by 1h8fulkat** [mealie-recipes/mealie#1253] [Feature Request]
I was thinking it might be nice to be able to automatically create meal plans by category and tag.  For example, I may have a dinner category that want to create meal plans off of but only want dinner recipes with tags matching "easy" or "quick" or a combination of other tags.

**Comment by hay-kot** [comment]
Hey! We've implemented a feature similar to this in v1

https://beta.mealie.io/group/mealplan/settings

**Reply by 1h8fulkat** [reply]
I'd say that's pretty close to what I was looking for, thanks!
